docs/UserGuide.md

```diff
@@ -139,7 +139,9 @@ Adds a student to the address book.
 
 ##### Constraints
 
-* **NAME** must only contain alphanumeric characters and spaces.
+* **NAME** must only contain alphanumeric characters and spaces. 
+* **NAME** is case-insensitive, i.e. you can use `jOhN Doe`
+  and we will help you to format it to become `John Doe`
 * **SCHEDULE** must be in the format of `DAY_OF_THE_WEEK`-`START_TIME`-`END_TIME`.
 * **DAY_OF_THE_WEEK** includes `Monday` `Tuesday` `Wednesday` `Thursday` `Friday` `Saturday` `Sunday`.
```

Comment:
can you help to indent line 146 and 147? I realised these two should be sub pointers for line 145. 
Also please help to change 146 to 
**DAY_OF_THE_WEEK** is one of `Monday` `Tuesday` `Wednesday` `Thursday` `Friday` `Saturday` `Sunday`.